docs/content/specs-defs/specs/shared/_index.md

```diff
@@ -1013,6 +1013,22 @@ Module owners **MAY** also have to provide additional outputs depending on the I
 
 <br>
 
+#### ID: RMFR8 - Category: Composition - End-of-life resource versions
```

Comment:
I think we need to cover the possibility that the entire resource is EoL. Rather than just configuration.

Reply:
Good point, shouldn't that be a separate requirement though? My thinking is that we should have a full-blown module deprecation/retirement process defined for this scenario. RMFR8 is not aiming for that all. I can take the action and start drafting something for the full module retirement scenario which we can then discuss internally first.

Reply:
Happy with this approach